test/contracts/staking/staking-factory.test.ts

```diff
@@ -8,9 +8,16 @@ import {before} from 'mocha'
 import {solidity} from 'ethereum-waffle'
 import {
     StakingPoolFactory,
-    ERC20PresetMinterPauser
+    ERC20PresetMinterPauser,
+    BondMediator,
```

Comment:
issue: unused imports

The linter is picking unused import, remove them please.